flake.nix

```diff
@@ -0,0 +1,46 @@
+{
+  description = "Nix development environment for LambdaClass Ethereum Consensus Client.";
+
+  # Flake inputs
+  inputs = {
+    nixpkgs.url      = "github:NixOS/nixpkgs/nixos-unstable";
```

Comment:
Why is this unstable?

Reply:
I understand that it is like the nightly version and also it was used in most tutorials. I can change it to the stable version if needed.

Reply:
I defer the decision to @Arkenan, but I'd prefer stable if none of the packages require otherwise.